project_generator/tools/uvision.py

```diff
@@ -69,6 +71,10 @@ class uVisionDefinitions():
             'Utilities': {
                 'Flash2': 'STLink\\ST-LINKIII-KEIL_SWO.dll',
             },
+            'OptxFile' : {
```

Comment:
yes. we use x prefix for uvision5, if this is common one, lets stay without x .

Yes, that was the idea, to provide dic per file, to know where goes what